### ISSUE TITLE
Missing config files for other methods

### ISSUE THREAD

**Issue by caiopetruccirosa** [paplhjak/Facial-Age-Estimation-Benchmark#20]
Hello! First of all, thank you very much for the paper.

I am trying to reproduce all the results reported in Table 6 of the paper for my research. However, after reviewing the YAML config files for the experiments, I found that there is only one base experiment provided for each benchmark. This does not cover all nine experiments listed in the sub-rows of Table 6 (Cross-Entropy, Regression, OR-CNN, DLDL, DLDL-v2, SORD, Mean-Var., Unimodal, FaRL + MLP).

From what I understand, the experiment config provided in the repository only corresponds to the ResNet50 model pre-trained on ImageNet with a Cross-Entropy loss head.

@paplhjak, could you please provide the experiment config files for all methods and weight initialization strategies (`IMDB-WIKI`, `ImageNet`, and `Random`) mentioned in the paper?

Thanks in advance!

**Comment by paplhjak** [comment]
> From what I understand, the experiment config provided in the repository only corresponds to the ResNet50 model pre-trained on ImageNet with a Cross-Entropy loss head.
Yes.

> @paplhjak, could you please provide the experiment config files for all methods and weight initialization strategies (IMDB-WIKI, ImageNet, and Random) mentioned in the paper?
I will look into it.

**Comment by caiopetruccirosa** [comment]
Thank you very much! I am looking forward to it.

**Comment by paplhjak** [comment]
I have some more pressing things to work on this week, but if all goes well, you can expect the files to get added next week.

**Comment by paplhjak** [comment]
The configuration files will be added by the end of the week.

**Comment by paplhjak** [comment]
The files can now be found in facebase/configs

**Comment by caiopetruccirosa** [comment]
Thank you very much!

**Comment by caiopetruccirosa** [comment]
Hi @paplhjak! Thanks for adding more config files!

However, I noticed that only the config files for the different weight initialization strategies of the baseline ResNet50 with a Cross-Entropy loss head experiments were added.

Could you also provide the config files for the other methods reported as implemented and evaluated in the paper, such as Regression, OR-CNN, DLDL, DLDL-v2, SORD, Mean-Var, Unimodal, and FaRL + MLP?

That would be greatly appreciated, as I am trying to reproduce all the results reported in the paper.
